lib/node-utils.ts

```diff
@@ -220,3 +224,29 @@ export function isRenderVariableDeclarator(
 
   return false;
 }
+
+// TODO: extract into types file?
+export type ModuleImportation =
```

Comment:
should it be

```suggestion
export type ModuleImport =
```

?

Importation sounds like spanglish (?)

Reply:
Good suggestion. I renamed that type so many times that I ended up with a shitty name 😂